test/unit/widgets/buttons.test.ts

```diff
@@ -105,7 +111,14 @@ describe('button', () => {
     const text = 'txt'
     const handler = () => {
     }
-    expect(button(dom, iconURI, text, handler)).toBeTruthy()
+    // FIXME: Not sure why this is needed, but
+    // JSS tries to do .insertBefore on a StyleSheet element,
+    // and that seems to fail in JSDOM (it works fine in the browser)
+    const domCopy = {
+      head: {},
+      createElement: dom.createElement.bind(dom)
+    }
+    expect(button(domCopy as unknown as HTMLDocument, iconURI, text, handler)).toBeTruthy()
```

Comment:
You can do some more testing on this, e.g. is the property class set? You can also make a spy for the handler, and manually trigger it using `button.dispatchEvent(new Event('click'))`.